### PR TITLE
c language grammar error

### PR DESCRIPTION
**Pull request by ose001004**
int ret,opt;->     int ret;
	                   int opt;

	/* Argument given? */

	optind -> int optind = -1;

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [3aa84a4afe282c8d990f779ab3252ecf4b103f78] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [3aa84a4afe282c8d990f779ab3252ecf4b103f78] - Code Rule Check OK.

**Comment by ose001004** [comment]
Thank you

File: apps/examples/smart_test/smart_test.c [hunk 1]
```diff
@@ -590,11 +590,12 @@ int main(int argc, FAR char *argv[])
...
 #endif
 {
-	int ret, opt;
+	int ret;
+	int opt;
 
 	/* Argument given? */
 
-	optind = -1;
+	int optind = -1;
```
Comment (sunghan-chang) [inline apps/examples/smart_test/smart_test.c]: This is a global variable which is defined at other file so that you must not add `int`.

Comment (shadaksharikp) [reply apps/examples/smart_test/smart_test.c]: i second Mr Sunghan opinion

**Comment by sunghan-chang** [comment]
@ose001004 This fix is not right. Let's close now, when you are free, please fix again and reopen this. Thank you for contribution.

**Pull request closed without merging**